core/src/main/java/com/boydti/fawe/config/Settings.java

```diff
@@ -107,6 +107,8 @@
                 " - History on disk or memory will be deleted",
         })
         public int MAX_HISTORY_MB = -1;
+        @Comment("Maximum time in milliseconds //calc can execute")
+        public int MAX_CALC_MS = 50;
```

Comment:
I think it would be better to call it `MAX_EXPRESSION_MS` and use it for all expressions instead of just calc.